### PR TITLE
Make title error visible and wrap when content is long

### PR DESCRIPTION
**Pull request by lagartoverde**
Solves https://binnenland.atlassian.net/browse/GN-2674

### CODE REVIEW

File: app/components/editor-document-title.hbs [hunk 1]
```diff
@@ -10,15 +10,27 @@
...
-          <input
-            class="au-c-app-chrome__title-input"
-            placeholder="Naamloos document"
-            id={{id}}
-            type="text"
-            value={{this.title}}
-            {{on "input" this.setTitle}}
-            {{did-insert this.focus}}
-          />
+          {{#if this.error}}
```
Comment (abeforgit) [inline app/components/editor-document-title.hbs:13]: You can put the if statement inside the class value here

File: app/components/editor-document-title.hbs [hunk 2]
```diff
@@ -10,15 +10,27 @@
...
+            <input
+              class="au-c-app-chrome__title-input au-c-input--error "
+              placeholder="Naamloos document"
```
Comment (abeforgit) [inline app/components/editor-document-title.hbs:16]: i18n please

**Review by abeforgit — CHANGES_REQUESTED** [review]

File: app/components/editor-document-title.hbs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {{#if @readOnly}}
   <h1 class="au-c-app-chrome__title">
-    {{this.title}}
+    {{limit-content this.title 70}}
```
Comment (abeforgit) [inline app/components/editor-document-title.hbs:3]: probably want some way to see the full title when read-only. (when not read-only, you can see it by editing it)

**Review by abeforgit — CHANGES_REQUESTED** [review]

**Review by elpoelma — APPROVED** [review]
Works as expected

**Pull request merged**